src/assets/components/index.web.js

```diff
@@ -0,0 +1,36 @@
+const outline = require('./outline').default;
+
+const components = {
+  get outline() {
+    return outline;
+  },
+  get arrowDown() {
+    return {uri: require('./arrowDown.png'), dimensions: {width: 16, height: 16}};
```

Comment:
Is this object ({uri, dimensions}) matches React Image component source props type?